defaults/severities.yaml

```diff
@@ -87,3 +87,4 @@ DisruptionEvent: ERROR
 DbEventsFilter: NORMAL
 EventsFilter: NORMAL
 EventsSeverityChangerFilter: NORMAL
+NodetoolEvent: ERROR
```

Comment:
@enaydanov I don't remember the logic exactly, if I set the default severity NORMAL and in the code we have dynamic severity that in some cases we increase it to ERROR, will it work?

Reply:
yes, it will work (when you use the `EventsSeverityChangerFilter` to change the default severity of an event)

Reply:
In the `defaults/severities.yaml` file we define worst severity.
I am not sure how it will work in your example

Reply:
As @juliayakovlev said, you define highest possible priority here (but you can override it by config option)

In the code you can use any severity which lower or equal to this.